meson.build

```diff
@@ -422,7 +422,7 @@ need_glx_functions = false
 if ['auto', 'gl'].contains(gl_backend)
   gl_dep = dependency('gl', required: gl_backend == 'gl')
```

Comment:
```suggestion
  gl_dep = dependency('gl', method: 'pkg-config', required: gl_backend == 'gl')
```

Reply:
I'll try that.

Reply:
This is not enough, opengl was not found before and is not found with this change either.

BTW is `method: 'pkg-config'` supposed to work at all with MinGW.